lib/gridfinityUtils/binBodyGenerator.py

```diff
@@ -119,9 +119,19 @@ def createGridfinityBinBody(
             adsk.core.ValueInput.createByReal(BIN_LIP_CHAMFER),
             True)
         chamferFeatures.add(topLipChamferInput)
-
-        bottomCutoutFace = lipCutout.endFaces.item(0)
-        currentDepth += BIN_CONNECTION_RECESS_DEPTH
+        
+    
+    if input.isSolid:
+        #top lip bottom chamfer
+        bottomTopLipChamferInput = chamferFeatures.createInput2()
+        bottomTopLipChamferEdges = adsk.core.ObjectCollection.create()
+        bottomTopLipChamferEdges.add(faceUtils.getBottomHorizontalEdge(lipCutout.faces.item(0)))
+        bottomTopLipChamferInput.chamferEdgeSets.addEqualDistanceChamferEdgeSet(bottomTopLipChamferEdges,
+            adsk.core.ValueInput.createByReal(.07),
+            True)
+        chamferFeatures.add(bottomTopLipChamferInput)
```

Comment:
that differs from how lip looks for an empty bin, what's the purpose of the extra fillet here?

Reply:
This is the extra .7 chamfer on the top, not really important to connection but does add some aesthetic and just more alignment to spec.  It could be included for a empty but not necessary and actually complicates print from overhangs.

![image](https://user-images.githubusercontent.com/95549855/224507942-60a59fc3-fbd2-40b7-b9bd-d92829b1ff8f.png)